Models/SSBPopulationStatistics/GraphQLTypes/LabeledValueType.cs

```diff
@@ -0,0 +1,14 @@
+using Backend.Models.SSBPopulationStatistics.POCO;
+using GraphQL.Types;
+
+namespace Backend.Models.SSBPopulationStatistics.GraphQLTypes
+{
+    public class LabeledValueType : ObjectGraphType<LabeledValue>
```

Comment:
This naming seems sort of unclear for the untrained eye, perhaps add msdocs on this?